README.md

```diff
@@ -93,6 +93,26 @@ Building clang from sources takes time and resources and it can be avoided:
   moment, see [KhronosGroup/SPIRV-LLVM-Translator#477](https://github.com/KhronosGroup/SPIRV-LLVM-Translator/issues/477)
   to track progress, discuss and contribute.
 
+## Configuring SPIR-V Headers
+
+The translator build is dependent on the official Khronos header file
+`spirv.hpp` that maps SPIR-V extensions, decorations, instructions,
+etc. onto numeric tokens. The official header version is available at
+[KhronosGroup/SPIRV-Headers](https://github.com/KhronosGroup/SPIRV-Headers).
+There are several options for accessing the header file:
+- By default, the header file repository will be downloaded from
+Khronos Group GitHub and put into `<build_dir>/SPIRV-Headers` (for
+out-of-tree builds) or into `llvm/projects` (for in-tree builds).
+- If you are building the translator in-tree, you can manually
+  download the SPIR-V Headers repo into `llvm/projects` - this
+  location will be automatically picked up by the translator build
+  scripts.
+- Any build type can also use an external installation of SPIR-V
+  Headers - if you have the headers downloaded somewhere in your
+  system and want to use that version, simply extend your CMake
+  command with
+  `-DLLVM_EXTERNAL_SPIRV_HEADERS_SOURCE_DIR=</path/to/headers_dir>`.
```

Comment:
Technically, there is difference between putting a directory into `llvm/projects` directory and setting `LLVM_EXTERNAL_SPIRV_HEADERS_SOURCE_DIR` yourself - former option likely to enable _build_ of the subproject by default (if there are `CMakeLists.txt`) while the latter requires `LLVM_EXTERNAL_PROJECTS_ENABLE` to be also set in order to build the project.

More details about how `llvm/project` works you can find in documentation to `add_llvm_subdirectory`  I linked above

Reply:
Updated the doc.